pkg/storage/s3/s3.go

```diff
@@ -998,3 +999,76 @@ func sharedCredentialsDataFromStaticCreds(accessKey, accessSecret string) []byte
 
 	return buf.Bytes()
 }
+
+// PutStorageTags is for adding/overwriting tags of the S3 bucket
+// which name is obtained using the ID() method.
+func (d *driver) PutStorageTags(tagList map[string]string) error {
```

Comment:
nit: It's misleading to call a map variable `tagList` (because it's not a list/slice of tags). We also don't want to repeat the type name in the variable name, so `tagMap` isn't a good choice either.

How about renaming `tagList` to `userTags` ? `desiredTags` is simply `tags` are also good choices.